ykrt/src/compile/jitc_yk/codegen/x86_64.rs

```diff
@@ -13,19 +13,22 @@ use super::{
     reg_alloc::{LocalAlloc, RegisterAllocator, StackDirection},
     CodeGen,
 };
+use crate::aotsmp::AOT_STACKMAPS;
 #[cfg(any(debug_assertions, test))]
 use crate::compile::jitc_yk::jit_ir::JitIRDisplay;
 use crate::compile::CompiledTrace;
+use crate::print_jit_state;
```

Comment:
It's generally nicer to collapse these into `use crate::{aotsmp::AOT_STACKMAPS, print_jit_state}` and so on. Doesn't have to be done here, but for future PRs.

Reply:
[2306aa1](https://github.com/ykjit/yk/pull/1061/commits/2306aa1f837ce7816a4a8d35a76e146c595f90ee)